### DISCUSSION TITLE
AnyCable go gives RPC error

### DISCUSSION THREAD

**Discussion by akashsuresh** [anycable/anycable#138] [General]
I got this error on my any cable deployment.

Perform error: rpc error: code = Canceled desc = Cancelled

After this error occurred I was unable to perform any actions on the socket until I restarted the Anycable go and any cable rpc service.

Any reason why this could have occurred

I am on rails 6.1.4 on ruby 3.0.0

grpc version 1.38.0

**Comment by palkan** [comment]
Interesting. According to [GRPC docs](https://grpc.github.io/grpc/core/md_doc_statuscodes.html), Canceled code is used when a caller cancels the operation. But we do not do that. From a quick search, I found that `canceled` could also happen in case underlying network problem occurred during the call execution.

>  I was unable to perform any actions on the socket

A single socket or all newly connected sockets as well?
